### PR TITLE
MISUV-7895 | Changed payment mainTransaction to 0060

### PR DESCRIPTION
**Pull request by owen-gilfellon**

### CODE REVIEW

**Comment by avelx** [comment]
@owen-gilfellon  - ideally if we can add ITs test as well, to have more robust coverage around this issue, but really up to you;

**Review by avelx — DISMISSED** [review]
approved with comment;

File: it/test/testConstants/IncomeSourceIntegrationTestConstants.scala [hunk 1]
```diff
@@ -348,6 +351,50 @@ object IncomeSourceIntegrationTestConstants {
...
+      "paymentLotItem" -> "000001",
+    )
+  }
+
+  private def financialDetailsPayment( transactionId: String,
+                                       amount: BigDecimal,
+                                       clearingDate: LocalDate,
+                                       clearingSapDocument: String): JsObject = {
+    Json.obj(
+      "taxYear" -> s"$testTaxYear",
```
Comment (avelx) [inline it/test/testConstants/IncomeSourceIntegrationTestConstants.scala:378]: I don't think we need to use string interpolation here

**Review by avelx — APPROVED** [review]

File: it/test/testConstants/IncomeSourceIntegrationTestConstants.scala (hunk 1 above)
Comment (owen-gilfellon) [reply it/test/testConstants/IncomeSourceIntegrationTestConstants.scala:378]: In FinancialDetail, taxYear is is String (Int in DocumentDetail)

**Pull request merged**